core/blockly.ts

```diff
@@ -626,7 +625,6 @@ export {BasicCursor};
 export {Block};
 export {BlocklyOptions};
 export {BlockDragger};
-export {BlockDragSurfaceSvg};
```

Comment:
Technically a breaking change.

Reply:
Yeah, and probably more importantly the signature of the constructor for WorkspaceSvg changed since I removed the drag surface args. I added the breaking change label to the PR accordingly.